include/swift/AST/Attr.def

```diff
@@ -414,6 +414,15 @@ DECL_ATTR(_projectedValueProperty, ProjectedValueProperty,
   OnVar | UserInaccessible,
   89)
 
+SIMPLE_DECL_ATTR(_hasMissingDesignatedInits, HasMissingDesignatedInits,
+  OnClass | UserInaccessible,
+  90)
+
+SIMPLE_DECL_ATTR(_inheritsSuperclassInitializers,
+  InheritsSuperclassInitializers,
```

Comment:
This should really be `InheritsSuperclassConvenienceInitializers`, or just `InheritsConvenienceInitializers`. (We already print inherited designated initializers, right?)